src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+    private String color;
```

Comment:
Remove the 'color' field in Circle class, as it is already present in the parent Figure class. Use the existing field to avoid redundancy. 🔄